index.html

```diff
@@ -86,7 +86,7 @@ <h1 class="section-header">How to Use</h1>
             <div class="instructions">
               <div class="text">
                 You can download the CSS file
-                <a href="https://github.com/shahednasser/sbuttons/blob/master/dist/sbuttons.min.css" class="cssLink"
+                <a href="#" id="downloadGithubRawHowTo" class="download-link" class="cssLink"
```

Comment:
can you remove `class="download-link"` as it already has class `cssLink`

Reply:
Oh yeah sorry for the mistake. i'll fix that